app/controllers/about_controller.rb

```diff
@@ -0,0 +1,5 @@
+class AboutController < ApplicationController
```

Comment:
you can name those controller - StaticPages, and they you can have those
```
def about
  render ....
end
```

more generic way and will fit to other static pages if needed later